core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/StreamUtil.java

```diff
@@ -165,6 +168,43 @@ public static InputStream opAuths(final Class clazz, final boolean logErrors) {
         return schemas;
     }
 
+
+    public static InputStream[] openStream(final String... urls) throws IOException {
+        final InputStream[] rtn = new InputStream[urls.length];
+        int count = 0;
+        for (final String url : urls) {
+            rtn[count++] = openStream(url);
```

Comment:
If this throws an exception then you need to ensure the rtn InputStreams are closed.